README.md

```diff
@@ -0,0 +1,4 @@
+# Wireless Telemetry Server
+Wireless telemetry server for Purdue Electric racing.
+Runs on a Raspberry Pi with a connected CANDapter and wireless router.
+[Wiki page](http://purdueelectricracing.com/wiki/index.php/Wireless_Telemetry)
```

Comment:
Wiki page looks great, thanks for good documentation on this.

Reply:
Seconded!